docs/rules/expect.md

```diff
@@ -0,0 +1,141 @@
+# expect-type/expect
+
+💼 This rule is enabled in the ✅ `recommended` config.
+
+🔧 This rule is automatically fixable by the [`--fix` CLI option](https://eslint.org/docs/latest/user-guide/command-line-interface#--fix).
+
+💭 This rule requires type information.
+
+<!-- end auto-generated rule header -->
+
+Enforces that types indicated in special comments match the types of code values.
+
+## Comment Types
+
+The following kinds of comments are supported:
+
+<!-- Markdownlint doesn't seem to understand the heading IDs... -->
+<!-- markdownlint-disable link-fragments -->
+
+- [`^?`](#twoslash)
+- [`$ExpectType`](#expecttype)
+- [`$ExpectTypeSnapshot`](#expecttypesnapshot)
+- [`$ExpectError`](#expecterror)
+
+<!-- markdownlint-enable link-fragments -->
+
+### Twoslash Syntax (`^?`)
+
+[Twoslash](https://www.typescriptlang.org/dev/twoslash) is the term for special comment directives in TypeScript code for tools such as this one.
+This plugin supports "twoslash assertions", which are comments containing `^?` and a type below a value in code to check that the value is that type.
```

Comment:
Consolidate these two paragraphs (I actually like the old version of this, which is shorter).

Reply:
Yeah, you're right - I'm going back to the old version mostly 😄